logfire/_internal/cli.py

```diff
@@ -388,12 +388,14 @@ def parse_info(_args: argparse.Namespace) -> None:
     related_packages: list[tuple[int, str, str]] = []
 
     for dist in importlib_metadata.distributions():
-        name = dist.metadata['Name']
+        metadata = dist.metadata
+        name = metadata.get('Name', '')
```

Comment:
`dist.metadata` was an 'empty' metadata object in some cases, so `dist.metadata['Name']` was giving a deprecation warning about missing keys which the tests translated to an error, and the warning said that it would eventually be a KeyError. So the fix is to not assume that `Name` and `Version` exist.